Packs/CommonScripts/TestPlaybooks/playbook-Test_URL_Regex.yml

```diff
@@ -43,7 +43,7 @@ tasks:
       - "2"
     scriptarguments:
       value:
-        simple: http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-2540
+        simple: http://cve.mitre.org/cgi-bin/cvename.cgi?name=cve-2008-2540
```

Comment:
why did this need to be changed?

Reply:
If I understand correctly this test prints a URL and then checks if it was picked up as an indicator. Our indicators are lowercase only and that's why the test failed.